### PR TITLE
Patch for issue #279

### PR DESCRIPTION
**Pull request by aka001**
I have quoted the directory path so that no further problem will be shown while running the command for directories containing spaces.
It relates to issue #279 9

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -135,7 +135,7 @@ deps: venv
 	fi
 	@echo "Making sure the latest version of pip is available"
 	# $(PIP) install -U pip
-	$(PIP) install --no-index --no-dependencies --find-links file:///$(CACHE)/python -r requirements.txt
+	$(PIP) install --no-index --no-dependencies --find-links file:///"$(CACHE)"/python -r requirements.txt
```
Comment (mitechie) [inline Makefile]: I think the quotes here need to go around the /python as well to best work.

So it'd be "file:///$(CACHE)/python"

**Comment by mitechie** [comment]
Thanks so much for the pull request. I think there's one small tweak to get the fix as readable as possible.

**Comment by aka001** [comment]
Thank you for the review. Shall I make a new pull request?

**Comment by mitechie** [comment]
Thanks so much for the update. I'll QA it later today and we'll get this landed.

**Comment by mitechie** [comment]
Mention #279 to make sure it's tied to the issue.

**Comment by aka001** [comment]
Yaa I have mentioned in the title.

**Comment by mitechie** [comment]
This required some more tweaks to work properly. The updated diff is at https://github.com/bookieio/Bookie/commit/7554f8265428dec57e945eb50ef1407694bf51fe but a fresh install in a directory with a space should work correctly now. 

Please look into the notes and other changes in there.

**Comment by mitechie** [comment]
Merged at 7554f8265428dec5

**Comment by aka001** [comment]
Okay got you. Thanks for merging it. :)

**Pull request closed without merging**